nixos/modules/services/web-servers/varnish/default.nix

```diff
@@ -7,32 +7,48 @@ with lib;
 {
   options = {
     services.varnish = {
-      enable = mkOption {
-        default = false;
-        description = "
-          Enable the Varnish Server.
-        ";
-      };
+      enable = mkEnableOption "Varnish Server";
 
       http_address = mkOption {
```

Comment:
17.03 also, it was already broken in March: https://botbot.me/freenode/nixos/search/?q=varnish